### PR TITLE
enhancement: add PR, bug report and feature request template

### PR DESCRIPTION
**Pull request by pratikjagrut**
Add github templates

### CODE REVIEW

**Comment by FabianKramm** [comment]
@LukasGentele should we include these?

**Comment by matskiv** [comment]
In the bug report, I would add a field for "Kubernetes distribution and version of the host cluster".

Do we need a Question type of issue too? Those are somewhat frequent. 
And if we do add a question template, I would maybe mention there that we have a Slack instance where they might get an answer quicker.

**Review by LukasGentele — CHANGES_REQUESTED** [review]
@FabianKramm I think this is a great idea. We should see how this goes and potentially add them to loft-sh/devspace repo as well. @pratikjagrut good job!

File: .github/ISSUE_TEMPLATE/feature-request.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+    attributes:
+      label: **Which solution do you suggest?**  
+    validations:
+      required: true
+
+  - type: textarea
+    id: alternateSolution
+    attributes:
+      label: **Which alternative solutions exist?**  
+    validations:
```
[the remark this answers is not in the document]
Comment (pratikjagrut) [reply .github/ISSUE_TEMPLATE/feature-request.md:23]: Good catch. Thank you.

File: .github/ISSUE_TEMPLATE/feature-request.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
+name: Feature Request
+description:  Suggest an idea for vcluster
+labels: kind/feature
+body:
+  - type: textarea
+    id: feature
+    attributes:
+      label: **Is your feature request related to a problem?**
```
[the remark this answers is not in the document]
Comment (pratikjagrut) [reply .github/ISSUE_TEMPLATE/feature-request.md:8]: Oh yes, my bad.

**Comment by pratikjagrut** [comment]
> In the bug report, I would add a field for "Kubernetes distribution and version of the host cluster".
> 
> Do we need a Question type of issue too? Those are somewhat frequent. And if we do add a question template, I would maybe mention there that we have a Slack instance where they might get an answer quicker.

@matskiv Thank you for the suggestions. I added a slack community link and the question you mentioned.

**Comment by pratikjagrut** [comment]
> @FabianKramm I think this is a great idea. We should see how this goes and potentially add them to loft-sh/devspace repo as well. @pratikjagrut good job!
> 

@LukasGentele Thank you :) We have a basic template in the devspace repo. Here I tried to replicate the templates from the Kubernetes repo. So if these templates are accepted here then we can modify templates from the devspace repo too.

**Review by FabianKramm — APPROVED** [review]
@pratikjagrut thanks for the PR! LGTM!

**Comment by pratikjagrut** [comment]
@LukasGentele Does this PR looks good to you?

**Review by LukasGentele — APPROVED** [review]
Looks great. Love it. Thanks!

**Pull request merged**